### PR TITLE
data: ZeeMonk-pixel

### PR DESCRIPTION
**Pull request by ZeeMonk-pixel**

### CODE REVIEW

**Review by thisisdishang — APPROVED** [review]

File: data/ZeeMonk-pixel.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+	}],
+	"links": [{
+			"name": "Follow me on Twitter",
+			"url": "https://twitter.com/MonkG_I",
+			"icon": "FaTwitter"
+		},
+		{
+			"name": "Linkedin",
+			"url": "https://linkedin.com/in/zeemonk1",
+			"icon": "FaTwitter"
```
Comment (Pradumnasaraf) [inline data/ZeeMonk-pixel.json:18]:
```suggestion
			"icon": "FaLinkedin"
```

**Review by Pradumnasaraf — CHANGES_REQUESTED** [review]
Hey,

The icons seem wrong with the URL and name provided. I have added suggestions

**Comment by ZeeMonk-pixel** [comment]
thank you, i have updated changes made.

**Review by Pradumnasaraf — APPROVED** [review]
Looks good to merge.

**Pull request merged**